api_core/google/api_core/bidi.py

```diff
@@ -698,7 +702,9 @@ def stop(self):
             if self._thread is not None:
                 # Resume the thread to wake it up in case it is sleeping.
                 self.resume()
-                self._thread.join()
+                self._thread.join(1.0)
```

Comment:
It might be helpful to note what is going on here. I am concerned in the future it may be forgotten.